File: business/src/main/java/no/nav/sbl/dialogarena/soknadinnsending/business/arbeid/Arbeidsforhold.java
package no.nav.sbl.dialogarena.soknadinnsending.business.arbeid;

import no.nav.sbl.dialogarena.soknadinnsending.business.domain.dto.Land;

public class Arbeidsforhold {
    public String orgnr;
    public String arbridsgiverNavn;
    public Land land;
    public String fom;
    public String tom;
    public Long edagId;
    public Long fastStillingsprosent = 0L;
    public boolean variabelStillingsprosent;
    public boolean harFastStilling;
}

File: src/main/kotlin/no/nav/sosialhjelp/soknad/arbeid/domain/Arbeidsforhold.kt
package no.nav.sosialhjelp.soknad.arbeid.domain

import java.time.LocalDate

data class Arbeidsforhold(
    val orgnr: String?,
    val arbeidsgivernavn: String,
    val fom: LocalDate?,
    val tom: LocalDate?,
    val fastStillingsprosent: Long? = 0L,
    val harFastStilling: Boolean?,
)
